Projects/UsertInterfaces/DesignSystem/Sources/RadioButton/DMSRadioButton.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+public struct DMSRadioButton: View {
+    @Binding var isOn: Bool
+    @Environment(\.isEnabled) private var isEnabled: Bool
+
+    public init(
+        isOn: Binding<Bool> = .constant(false)
+    ) {
+        self._isOn = isOn
+    }
+
+    public var body: some View {
+        Button {
+            self.isOn.toggle()
+        } label: {
+            ZStack {
+                Circle()
+                    .fill(Color.PrimaryVariant.primary)
+                    .frame(width: 20, height: 20)
+                    .clipShape(Circle())
+                Circle()
+                    .fill(Color.white)
+                    .frame(width: 15, height: 15)
+                Circle()
+                    .fill(Color.PrimaryVariant.primary)
+                    .frame(width: 10, height: 10)
+            }.disabled(isEnabled)
```

Comment:
disabled여기에 안붙이여도 실제 사용할 때
```swift
DMSRadioButton(isOn: $isOn)
.disabled(true)
```
이런식으로 써서 사용되니 빼주셔도 될거같아요
```suggestion
            }
```